nomad/periodic.go

```diff
@@ -85,6 +87,7 @@ func (s *Server) DispatchJob(job *structs.Job) (*structs.Evaluation, error) {
 		return nil, err
 	}
 
+	// TODO: do the timestamps need to be at the same time that the raft index is updated?
```

Comment:
The timestamp is a rough approximation for when the eval was inserted as part of a raft transaction. 

Can remove this comment.